cmd/crossplane/rbac/rbac.go

```diff
@@ -38,17 +38,19 @@ const (
 
 // Command configuration for the RBAC manager.
 type Command struct {
-	Name             string
-	Sync             time.Duration
-	LeaderElection   bool
-	ManagementPolicy string
+	Name                string
+	Sync                time.Duration
+	LeaderElection      bool
+	ManagementPolicy    string
+	ProviderClusterRole string
 }
 
 // FromKingpin produces the RBAC manager command from a Kingpin command.
 func FromKingpin(cmd *kingpin.CmdClause) *Command {
 	c := &Command{Name: cmd.FullCommand()}
 	cmd.Flag("sync", "Controller manager sync period duration such as 300ms, 1.5h or 2h45m").Short('s').Default("1h").DurationVar(&c.Sync)
 	cmd.Flag("manage", "RBAC management policy.").Short('m').Default(ManagementPolicyAll).EnumVar(&c.ManagementPolicy, ManagementPolicyAll, ManagementPolicyBasic)
+	cmd.Flag("provider-clusterrole", "A ClusterRole enumerating the permissions provider packages may request.").StringVar(&c.ProviderClusterRole)
```

Comment:
This is unset by default and thus all permission requests will be rejected. I'm thinking we should create a default (e.g. allowing secrets, events, and configmaps) via the Helm chart and pass that in here.